src/libraries/System.Private.CoreLib/src/ILLink/ILLink.Substitutions.NoArmIntrinsics.xml

```diff
@@ -1,52 +1,55 @@
 <linker>
   <assembly fullname="System.Private.CoreLib">
+    <type fullname="System.Runtime.Intrinsics.Vector64">
```

Comment:
The IL linker doesn't implicitly understand the `IsSupported` or `IsHardwareAccelerated` flags. `Vector64<T>` is only usable/accelerable when ARM intrinsics are supported and so we have to treat it as `false` otherwise.